specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1437,6 +1437,437 @@
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/NorthSouthHardeningsPolicyList"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/northSouthHardenings/{northSouthResourceName}/policies/{northSouthPolicyName}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get a north-south hardening policy": {
+                        "$ref": "./examples/NorthSouthHardenings/GetNorthSouthHardeningsPolicies_example.json"
+                    }
+                },
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "description": "Get a north-south hardening policy",
+                "operationId": "NorthSouthHardeningsPolicies_Get",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersion"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionId"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupName"
+                    },
+                    {
+                        "$ref": "#/parameters/NorthSouthResourceName"
+                    },
+                    {
+                        "$ref": "#/parameters/NorthSouthPolicyName"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/NorthSouthHardeningsPolicy"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "x-ms-examples": {
+                    "Put a north-south hardening policy": {
+                        "$ref": "./examples/NorthSouthHardenings/PutNorthSouthHardeningsPolicies_example.json"
+                    }
+                },
+                "tags": [
+                    "NorthSouthHardenings"
+                ],
+                "description": "Put a north-south hardening policy",
+                "operationId": "NorthSouthHardeningsPolicies_Put",

```

Comment:
If this operation is for creating or updating a policy resource then the operation id should be `NorthSouthHardeningsPolicies_CreateOrUpdate`

Reply:
Done